webapp/app.py

```diff
@@ -254,8 +255,11 @@ def utility_processor():
     view_func=appliance_install,
 )
 app.add_url_rule(
-    "/appliance/portfolio",
-    view_func=appliance_portfolio,
+    "/appliance/portfolio", view_func=appliance_portfolio,
+)
+
+app.add_url_rule(
+    "/static/files/meta_image.png", view_func=share_meta_image, methods=["GET"]
```

Comment:
It's misleading to put this in `/static/files` when it's not a static endpoint. Why wouldn't this be `ubuntu.com/generate-meta-image` or something?

Reply:
@nottrobin no reason, I wasn't 100% sure what I was doing at first... as long as the headers are right and it is cacheable, I don't think it matters?

Reply:
No indeed. Just thought I'd mention it 🙂